xcodeproj/internal/input_files_aspect.bzl

```diff
@@ -124,6 +124,7 @@ def _input_files_aspect_impl(target, ctx):
     non_arc_srcs = []
     other = []
 
+    # buildifier: disable=uninitialized
```

Comment:
This one is a bug btw. If we rename the arguments it works. It's incorrectly looking at the variables declared later...

Not that we have to address that here, just commenting.

Reply:
Ah. I wasn't sure. I have not used embedded functions in Starlark.